src/github.com/matrix-org/go-neb/api.go

```diff
@@ -228,3 +229,37 @@ func (s *configureServiceHandler) OnIncomingRequest(req *http.Request) (interfac
 		NewConfig types.Service
 	}{body.ID, body.Type, oldService, service}, nil
 }
+
+type getServiceHandler struct {
+	db *database.ServiceDB
+}
+
+func (h *getServiceHandler) OnIncomingRequest(req *http.Request) (interface{}, *errors.HTTPError) {
+	if req.Method != "GET" {
```

Comment:
The most obvious objection being that the go http client doesn't support without building the request by hand.